wallet/src/de/schildbach/wallet/ui/InviteHandlerActivity.kt

```diff
@@ -36,7 +36,11 @@ class InviteHandlerActivity : AppCompatActivity() {
         private const val EXTRA_SILENT_MODE = "extra_silent_mode"
 
         @JvmStatic
-        fun createIntent(context: Context, invite: InvitationLinkData, silentMode: Boolean): Intent {
+        fun createIntent(
+            context: Context,
+            invite: InvitationLinkData,
+            silentMode: Boolean
+        ): Intent {
```

Comment:
I noticed that the latest version of Android Studio automatically format methods this way. This is fine by me, but would like to highlight it, so that we are all aware of this new "feature" of AS.

Reply:
looks like ktlint or something.  Perhaps the line length is too long.

I have been experimenting more with ktlint and detekt in a branch in dpp.

https://github.com/dashevo/android-dpp/pull/23/checks?check_run_id=3318478874#step:3:246 (this link shows the detekt report).